po/it.po

```diff
@@ -0,0 +1,2257 @@
+#
+# Cvfosammmm <r.griesel@gmail.com>, 2020.
+# Kowalski7cc <kowalski.7cc@gmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: setzer\n"
+"Report-Msgid-Bugs-To: \n"
```

Comment:
Link is missing

Reply:
It's not set also in the German translation too

Reply:
Good catch, @cvfosammmm, but it should be added anyway.

Reply:
Btw I'm updating them for the latest commits